src/tests/resource.test.ts

```diff
@@ -15,19 +15,28 @@ describe('ResourceTests', () => {
     })
 
     test('Create resource', async () => {
-        const res = await api.resources.create({
-            name: 'testResc',
+        const res1 = await api.resources.create({
+            name: 'testParentResc',
             type: 'unixfilesystem',
             host: 'ip-172-31-12-211',
```

Comment:
The hostname assigned to `host` is going to be an issue the minute it doesn't resolve. Open an issue for revisiting the hostnames used in the tests.

Reply:
Please add a comment here referencing the issue that was created.

Reply:
Done.